Mathlib/RingTheory/DedekindDomain/Dvr.lean

```diff
@@ -143,3 +141,38 @@ theorem IsDedekindDomain.isDedekindDomainDvr [IsDedekindDomain A] : IsDedekindDo
   { isNoetherianRing := IsDedekindRing.toIsNoetherian
     is_dvr_at_nonzero_prime := fun _ hP _ =>
       IsLocalization.AtPrime.discreteValuationRing_of_dedekind_domain A hP _ }
+
+variable {A}
+
+theorem IsDedekindDomainDvr.ring_dimensionLEOne (h : IsDedekindDomainDvr A) :
+    Ring.DimensionLEOne A where
+  maximalOfPrime := by
+    intro p hp hpp
+    rcases p.exists_le_maximal (Ideal.IsPrime.ne_top hpp) with ⟨q, hq, hpq⟩
+    let f := (IsLocalization.orderIsoOfPrime q.primeCompl (Localization.AtPrime q)).symm
+    let P := f ⟨p, hpp, HasSubset.Subset.disjoint_compl_left hpq⟩
+    let Q := f ⟨q, hq.isPrime, HasSubset.Subset.disjoint_compl_left (fun _ a => a)⟩
```

Comment:
These `HasSubset.Subset` names are only for dot notation; either use them that way or use the theorem they're aliases for, please